app/views/merchants/invoices/show.html.erb

```diff
@@ -5,19 +5,19 @@
 <center><p>Invoice #<%= @invoice.id %></p></center>
 
 <p>Status: <%= @invoice.status %></p>
-<p>Created on: <%= @invoice.created_at.strftime("%A, %B %-d, %Y") %></p>
-<p>Total Revenue: <%= @total_revenue %></p>
+<p>Created on: <%= @invoice.formatted_time %></p>
+<p>Total Revenue: <%= number_to_currency(@invoice.invoice_total_revenue / 100) %></p>
```

Comment:
Does this work if the resulting number has trailing zeros? Might need a `100.0` or a `precision: 2`. Something to do later to make your testing more robust.

Reply:
Good call on dividing by 100!  We should maybe consider adding a helper method to do that for us so we don't have to think about it

Reply:
I think it does take care of that precision but good call to test for it!